src/components/MenuItem.js

```diff
@@ -79,7 +79,6 @@ const MenuItem = (props) => {
             props.icon ? styles.ml3 : undefined,
             props.shouldShowBasicTitle ? undefined : styles.textStrong,
             props.interactive && props.disabled ? {...styles.disabledText, ...styles.userSelectNone} : undefined,
-            styles.pre,
```

Comment:
Looks like you have forgotten to undo the change here.

Reply:
@sobitneupane Oops. Fixed. Thanks.